CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## [3.7.0](https://github.com/marklogic-community/mlxprs/releases/tag/v3.7.0)
+## [3.7.0](https://github.com/marklogic/mlxprs/releases/tag/v3.7.0)
 
 - [#DEV-261](https://project.marklogic.com/jira/browse/DEVEXP-261) Submit an Optic DSL Query to /v1/rows and get back JSON
```

Comment:
Unrelated - I don't think we should put JIRA URLs in the changelog - they're meaningless to external users. I wouldn't even bother with the JIRA ID. Just the description of the feature is enough.